.craft.yml

```diff
@@ -30,12 +30,14 @@ targets:
     bucket: sentry-sdk-assets
     paths:
       - path: /sentry-cli/{{version}}/
-        maxCacheAge: 2592000
+        metadata:
+          cacheControl: 'public, max-age=2592000'
```

Comment:
It's already a bit late for this, but shouldn't this be a setting configured more centrally in GCS rather than in each repository? Not that we're going to make many changes on this, but it feels wrong (and especially too verbose) to have this here.

Reply:
Different projects (and more interestingly, different paths in the same bucket) may have different metadata values. I don't see a nicer way to store this kind of information anywhere else without confusing people who push new releases.